gallery/src/styles/NxList/NxListPage.tsx

```diff
@@ -124,6 +124,32 @@ const NxListPage = () =>
               would only be a single list item which contains the error alert.
             </td>
           </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-list__item--clickable</code></td>
+            <td className="nx-cell"/>
+            <td className="nx-cell">Modifier of <code className="nx-code">.nx-list__item</code></td>
+            <td className="nx-cell">
+              Applied to <code className="nx-code">.nx-list__item</code> this allows for the correct styling of
+              clickable lists. Ensure that the <code className="nx-code">.nx-list</code> modifier
+              {' '}<code className="nx-code">.nx-list--clickable</code> is also used.
```

Comment:
Should probably elaborate on how it must contain a single `.nx-list__btn` or `.nx-list__link` as a child

Reply:
> Should probably elaborate on how it must contain a single `.nx-list__btn` or `.nx-list__link` as a child

https://github.com/sonatype/sonatype-react-shared-components/pull/226/commits/f54fbde52ee8d895cbb4b916ab17a599dfa4f056